packages/frontend/src/providers/AppProvider.tsx

```diff
@@ -188,12 +191,17 @@ export const AppProvider: FC = ({ children }) => {
 
     const activeJobStatusToast = useCallback(() => {
         if (activeJob) {
-            const toastTitle = `${
-                refreshStatusInfo(activeJob?.jobStatus).title
-            }`;
+            const toastTitle = `${jobStatusLabel(activeJob?.jobStatus).label}`;
             const hasSteps = !!activeJob.steps.length;
             switch (activeJob.jobStatus) {
                 case 'DONE':
+                    if (activeJob.jobType === JobType.CREATE_PROJECT) {
```

Comment:
if the active job was a `CREATE_PROJECT` job then refresh our view of the projects